#Descripcion de Variables que componen al Mensaje

uint8 state_code
string state_name
float64 yaw_err
float64 dist_err